src/callbacks/onResize.js

```diff
@@ -51,10 +51,10 @@ export default function onResize() {
         });
 
     //Annotate grouping.
-    if (this.config.y.grouping) annotateGrouping.call(this);
```

Comment:
Confirm that removing the if() statements is intentional.

Reply:
Yeah, just moved the logic to the imports themselves.